running/SELinuxEscape/podman.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+# podmah.sh demo script.
+# This script will demonstrate a lot of the features of podman, concentrating
+# on the security features.
+
+# Setting up some colors for helping read the demo output.
+# Comment out any of the below to turn off that color.
+bold=$(tput bold)
+blue=$(tput setaf 4)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${blue}$1${reset}"
+}
```

Comment:
non-blocking idea: we could refactor those helpers into a common file shared across demos (and document it the main README.md).

Reply:
Yes, I plan on doing that next.

Reply:
Honest, I didn't see these comments before my prior!  Poor @vrothberg, here only a few months and already he's starting to think like a Sweeney....